src/uu/cut/src/cut.rs

```diff
@@ -111,8 +118,13 @@ struct Options {
     zero_terminated: bool,
 }
 
+enum Delimiter {
+    Whitespace,
+    String(String), // one char long, String because of UTF8 representation
```

Comment:
I know this comment was here before and we shouldn't fix it in this PR, but this is nonsense right? Looks like somebody got confused between a C `char` (i.e. `u8`) and a Rust char, which is actually a full UTF-8 character.

Reply:
good catch!